app/src/main/res/raw/rooms_info.json

```diff
@@ -3,6 +3,7 @@
     {
       "roomCode": "VL",
       "floorCode": "VL-1",
+      "Campus": "LOY",     
```

Comment:
will the fact that campus is with a capital "C" affect anything?

Reply:
No it is with a capital C in Place.java this is why I had to put in like this.. so it can add it to the place using inheritance

Reply:
I'm not sure it was necessary to add the campus attribute to the rooms object because you can access that from the floor object that you can get using the floor code